node-notes-cli/app.js

```diff
@@ -0,0 +1,13 @@
+import readNotes from './read.js';
+import addNote from './add.js';
+import deleteNote from './delete.js';
+import updateNote from './update.js';
+
+const [,, feature, ...args] = process.argv;
+switch (feature) {
+  case 'read': readNotes(1); break;
```

Comment:
Having gone and read the `readNotes` code, I see you intend to pass a boolean. It's best to use `true` and `false` for the booleans instead of `1` and `0`. It's much more readable that way.